src/core/ddsi/src/q_radmin.c

```diff
@@ -303,9 +304,10 @@ static void nn_rbuf_release (struct nn_rbuf *rbuf);
 #define RMSGTRACE(...)              TRACE_CFG (rmsg, rmsg->chunk.rbuf->rbufpool->logcfg, __VA_ARGS__)
 #define RDATATRACE(rdata, ...)      TRACE_CFG ((rdata)->rmsg, (rdata)->rmsg->chunk.rbuf->rbufpool->logcfg, __VA_ARGS__)
 
-static uint32_t align8uint32 (uint32_t x)
+static uint32_t align_literal_uint32 (uint32_t x)
```

Comment:
It would be nice to have a why in here. "align_literal_uint32" doesn't make it clear that this is for aligning a `struct nn_rmsg`.

Reply:
Fair point, maybe `align8Puint32` would be better? 8 bytes or pointer, whichever is widest.

Anyway, I'm not known for consistently coming up with beautiful names, so I am not going to fuss over the name of some very locally used alignment function ...

Reply:
`align8Puint32` is correct but doesn’t make the code any more self-evident. Maybe `align_rmsg` or change it to be a general alignment function that takes as an argument a constant `ALIGNOF_RMSG`.

Reply:
This makes sense.  How about:

```
static uint32_t align_rmsg (uint32_t x)
{
  /* Align rmsg chunks to the larger of sizeof(void*) or 8.
  Ideally, we would use C11's alignof(void *), but sizeof(void *) is equivalent
  for primitive types. */
  const uint32_t align_size = sizeof(void *) > 8 ? sizeof(void *) : 8;
  return (x + align_size - 1) & -align_size;
}
```

and then returning to the original naming convention for the lhs of the assignment.  For example:

`uint32_t size8P = align_rmsg (size);`